.github/workflows/bundle-size.yml

```diff
@@ -0,0 +1,21 @@
+name: Compressed Size
+
+on:
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    build:
+        name: Bundle size check
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use desired version of NodeJS
+              uses: actions/setup-node@v2
+              with:
+                  node-version: '12'
```

Comment:
@fashuvo Instead of specifying a specific node version, we can make use of the version specified in `.nvmrc` file. So we are in sync with the node version of the project.

Here is a similar implementation - https://github.com/Automattic/woocommerce-payments/blob/5b9809465e0fb9ec3476d40f82b85daf82d512e9/.github/workflows/e2e-test.yml#L103-L107